src/components/StatusUpdater/index.tsx

```diff
@@ -114,7 +115,15 @@ const StatusUpdaterProtected = ({ activity, current, fullWidth, user, thisOrg }:
     setConfirming(true);
   }
 
-  const actions = getStatusOptions(current, activity.startTime, activity.forceStandbyOnly, activity.earlySignInWindow);
+  const lastOrgId = participant?.timeline[0].organizationId;
+  const currentOrgId = participant?.organizationId;
+  let actions;
+  if (isActive(current) && currentOrgId !== lastOrgId) {
+    const lastOrgName = activity.organizations[lastOrgId].rosterName ?? activity.organizations[lastOrgId].title;
+    actions = [{ id: 7, newStatus: ParticipantStatus.SignedOut, text: `Sign Out from ${lastOrgName}` }];
```

Comment:
When the sign out goes through, is it stamped as the correct org (their previous one)? I'm not seeing anything in the action specifying which org the action applies to, so presumably it would apply to their current org?

Reply:
hmm... that's a good call, let me look into that.

Reply:
fixed